explore/src/main/scala/explore/observationtree/TreeComp.scala

```diff
@@ -113,7 +113,7 @@
 //             setter
 //               .set(tree,
 //                    getSet.getter.get,
-//                    { value: props.treeMod.ElemWithIndex =>
+//                    { value: props.treeMod.ElemWithIndexOpt =>
```

Comment:
Change in commented code?

Reply:
If we ever uncomment it, we want that change ;)